src/modules/10-common/pages/login/LoginPage.tsx

```diff
@@ -72,10 +72,10 @@ const LoginPage: React.FC = () => {
       if (response.ok) {
         const json = await response.json()
 
-        AppStorage.set('token', json.resource.token)
-        AppStorage.set('acctId', json.resource.defaultAccountId)
-        AppStorage.set('uuid', json.resource.uuid)
-        AppStorage.set('lastTokenSetTime', +new Date())
+        SecureStorage.set('token', json.resource.token)
+        SecureStorage.set('acctId', json.resource.defaultAccountId)
+        SecureStorage.set('uuid', json.resource.uuid)
+        SecureStorage.set('lastTokenSetTime', +new Date())
```

Comment:
Existing code, but instead of casting to a number, you can just use `Date.now()`
```suggestion
        SecureStorage.set('lastTokenSetTime', Date.now())
```

Reply:
done